integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -141,6 +146,9 @@ def export(self, **kwargs):
                 if self.course_api is None:
                     self.course_api = CourseApiClient()
                 course_details = self.course_api.get_course_details(course_id)
+                LOGGER.info('[Integrated Channel] Successfully retrieved course details for course: {}'.format(
```

Comment:
maybe even log course_details if not None? We have been having trouble with course_details down the line, we can remove it in future if too much text

Reply:
we are in a loop here so prior value of course_details is also used if available, so we should probably log those